operator/builtin/input/file/config_test.go

```diff
@@ -80,6 +80,24 @@ func TestConfig(t *testing.T) {
 				return cfg
 			}(),
 		},
+		{
+			"include_glob_double_asterisk",
+			false,
+			func() *InputConfig {
+				cfg := defaultCfg()
+				cfg.Include = append(cfg.Include, "**.log")
+				return cfg
+			}(),
+		},
+		{
+			"include_glob_double_asterisk_nested",
+			false,
+			func() *InputConfig {
+				cfg := defaultCfg()
+				cfg.Include = append(cfg.Include, "directory/**/*.log")
```

Comment:
Could we add one more test for `**/directory/**/*.log`?

Reply:
Done